Gemfile

```diff
@@ -4,7 +4,7 @@ source 'https://rubygems.org'
 
 gemspec
 
-gem 'faraday', '~> 1.0', '>= 1.0.0'
+gem 'faraday', '>= 2.0.0', '< 3.0'
```

Comment:
Can you actually change this to `gem "faraday", ">= 2.0.1"`.

Long story short, 2.0.0 will cause a ton of problems that took me a whole day of debugging to figure out.